util/test/testUtil.h

```diff
@@ -44,7 +45,7 @@ void valuecheck(const Eigen::DenseBase<DerivedA>& a, const Eigen::DenseBase<Deri
 
 void valuecheck(double a, double b, double tolerance = 1e-8)
 {
-  if (abs(a - b) > tolerance) {
+  if (std::abs(a - b) > tolerance) {
```

Comment:
I'm fine with merging. But in my opinion testUtil.h would be a better location for valuecheck. Can I move it there and have other files that call it include testUtil.h?